src/app/app.component.html

```diff
@@ -0,0 +1,47 @@
+<!----<h1>{{title}}</h1>
```

Comment:
Do not comment code, it is not a good practice. if something is not required remove it.